### PR TITLE
Feat: Transfer Balance Validation

### PR DESCRIPTION
**Pull request by AlexZorkin**
- Added backend validation on transfer create and update to ensure Suppliers can't create transfers with a quantity greater than their compliance units balance
- Added a ref to BCAlert so it can render multiple times on click
- Fixed render bug on org user display
- Conditional rendering of supplier balance added to header component
- Formatting in supplier balance fixed

### CODE REVIEW

File: frontend/src/views/Transfers/AddEditViewTransfer.jsx [hunk 1]
```diff
@@ -210,14 +211,21 @@ export const AddEditViewTransfer = () => {
...
       }
+      alertRef.current.triggerAlert()
     },
     onError: (_error, _variables) => {
-      setAlertMessage(
-        transferId
-          ? t('transfer:actionMsgs.errorUpdateText')
-          : t('transfer:actionMsgs.errorCreateText')
-      )
+      const errorMsg = _error.response.data?.detail
```
Comment (prv-proton) [inline frontend/src/views/Transfers/AddEditViewTransfer.jsx:217]: would also need to clear the confirmation modal. As it still persisting when we click on "Sign and send" button.

File: backend/lcfs/web/api/transfer/validation.py [hunk 2]
```diff
@@ -18,7 +18,7 @@ def __init__(
         self.org_repo = org_repo
         self.request = request
 
-    async def update_transfer(self, request: Request, transfer_create: TransferCreateSchema):
+    async def governent_update_transfer(self, request: Request, transfer_create: TransferCreateSchema):
```
Comment (prv-proton) [inline backend/lcfs/web/api/transfer/validation.py:21]: just a typo on "government"

File: backend/lcfs/web/api/transfer/views.py [hunk 3]
```diff
@@ -33,14 +33,14 @@ async def get_transfer(
 @router.put("/{transfer_id}", response_model=TransferSchema, status_code=status.HTTP_200_OK)
 @view_handler
 @roles_required("Government")
-async def update_transfer(
+async def governent_update_transfer(
```
Comment (prv-proton) [inline backend/lcfs/web/api/transfer/views.py:36]: same as above, typo.

**Review by prv-proton — CHANGES_REQUESTED** [review]
All functionality looks good. Thank you.

File: frontend/src/views/Transfers/AddEditViewTransfer.jsx (hunk 1 above)
Comment (AlexZorkin) [reply frontend/src/views/Transfers/AddEditViewTransfer.jsx:218]: I added the modal close, as well as a scroll to top so the user can see the error.

**Pull request merged**